src/modules/catalog.js

```diff
@@ -472,100 +292,35 @@ class Catalog {
   /**
    * Retrieves item(s) from index for the given section or specific item ID
    *
-   * @function getItem
-   * @param {object} parameters - Additional parameters for item details
-   * @param {string} parameters.id - The ID of the item you'd like to retrieve
-   * @param {object} [networkParameters] - Parameters relevant to the network request
-   * @param {number} [networkParameters.timeout] - Request timeout (in milliseconds)
-   * @returns {Promise}
-   * @see https://docs.constructor.io/rest_api/items/get_items
-   */
-  getItem(parameters = {}, networkParameters = {}) {
-    const queryParams = {};
-    let requestUrl;
-    const fetch = (this.options && this.options.fetch) || nodeFetch;
-    const controller = new AbortController();
-    const { signal } = controller;
-
-    if (parameters) {
-      const { section } = parameters;
-
-      // Pull section from parameters
-      if (section) {
-        queryParams.section = section;
-      }
-    }
-
-    try {
-      if (parameters.id) {
-        requestUrl = createCatalogUrl(`item/${parameters.id}`, this.options, queryParams);
-      } else {
-        requestUrl = createCatalogUrl('item', this.options, queryParams);
-      }
-    } catch (e) {
-      return Promise.reject(e);
-    }
-
-    // Handle network timeout if specified
-    helpers.applyNetworkTimeout(this.options, networkParameters, controller);
-
-    return fetch(requestUrl, {
-      method: 'GET',
-      headers: {
-        'Content-Type': 'application/json',
-        ...helpers.createAuthHeader(this.options),
-      },
-      signal,
-    }).then((response) => {
-      if (response.ok) {
-        return response.json();
-      }
-
-      return helpers.throwHttpErrorFromResponse(new Error(), response);
-    }).then((json) => json);
-  }
-
-  /**
-   * Retrieves items from index for the given section
-   *
    * @function getItems
    * @param {object} parameters - Additional parameters for item details
-   * @param {string} parameters.section - The index section you'd like to retrieve results from
-   * @param {number} [parameters.num_results_per_page] - The number of items to return. Defaults to 20. Maximum value 1,000
-   * @param {number} [parameters.page] - The page of results to return. Defaults to 1
+   * @param {string} parameters.id - Id(s) of items to return. Maximum number of ids to request is 1000.
+   * @param {string} parameters.key - The API key of the index that you'd like to retrieve results from.
+   * @param {string} parameters.section - The index section you'd like to retrieve results from.
+   * @param {number} parameters.num_results_per_page - The number of items to return. Defaults to 100. Maximum value 100.
```

Comment:
Can we make this `numResultsPerPage`?

Reply:
this is how it's named also in other methods, should I rename it there too? @esezen